common-lib/src/main/scala/com/gu/mediaservice/lib/argo/model/Action.scala

```diff
@@ -10,11 +10,9 @@ import play.api.libs.functional.syntax._
 case class Action(name: String, href: URI, method: String)
 
 object Action {
-
   implicit val actionWrites: Writes[Action] = (
     (__ \ "name").write[String] ~
-      (__ \ "href").write[String].contramap((_: URI).toString) ~
-      (__ \ "method").write[String]
-    )(unlift(Action.unapply))
-
+    (__ \ "href").write[String].contramap((_: URI).toString) ~
```

Comment:
We could write an implicit that does this for URIs everywhere (I've seen it else where)...